wcfsetup/install/files/lib/util/ArrayUtil.class.php

```diff
@@ -81,7 +81,7 @@ public static function encodeHTML($array)
      * Applies stripslashes on all elements of the given array.
      *
      * @param array $array
-     * @return  array
+     * @return  array|string
```

Comment:
This one is not more correct than the existing one (because string may only be returned if `$array` is not an array) and much less useful.